### PR TITLE
add gcp openid documentation

### PR DESCRIPTION
**Pull request by jonbcampos-alto**
This is just documentation for the OpenID section, specifically for GCP/GKE users. Just documentation.

### CODE REVIEW

**Comment by jmazzitelli** [comment]
Here's the netlify link to review: https://deploy-preview-362--kiali.netlify.app/documentation/staging/configuration/authentication/openid/#_using_with_google_cloud_platform_gke_oauth2

**Comment by jonbcampos-alto** [comment]
esh, big images, let me shrink them down a touch

**Comment by jonbcampos-alto** [comment]
@jmazzitelli thank you for the link, I've resized the images, will the link update automatically?

**Comment by jmazzitelli** [comment]
> will the link update automatically?

yes

**Comment by jonbcampos-alto** [comment]
@jmazzitelli answer is yes, looks much nicer. thx

File: content/documentation/staging/configuration/authentication/openid.adoc [hunk 1]
```diff
@@ -292,3 +292,74 @@ your signing key for Kiali is not a standard size (16, 24 or 32 bytes long).
 Enabling the `Implicit Flow Enabled` option of the client will make the problem
 go away, but be aware that the implicit flow is less secure, and not
 recommended.
+
+=== Using with Google Cloud Platform / GKE OAuth2
+
+If you are already using Google Cloud Platform (GCP) and products such as
+Google Kubernetes Engine (GKE) then it is very easy to integrate with Kiali's
+OpenID.
```
Comment (israel-hdez) [inline content/documentation/staging/configuration/authentication/openid.adoc:300]:
```suggestion
If you are using Google Cloud Platform (GCP) and its products such as
Google Kubernetes Engine (GKE), it should be straightforward to configure Kiali's OpenID
strategy to authenticate using your Google credentials.
```

**Review by israel-hdez — COMMENTED** [review]
I have a small change in wording in the first paragraph.
Everything else looks quite fine!

**Comment by israel-hdez** [comment]
Thanks @jonbcampos-alto .
I'm merging. Your contribution will be available in Kiali website today.

**Pull request merged**